modules/specialized-guides/pages/salt/salt-terminology.adoc

```diff
@@ -70,6 +70,11 @@ They contain information about a client or group of clients.
 Pillars allow you to send confidential information to a targeted client or group of clients.
 Pillars are useful for sensitive data, configuration of clients, variables, and any arbitrary data.
 +
+[NOTE]
+====
+If you change pillar data on the server ({salt} master) the actual pillar data on the client (minion) is updated only after calling [literal]``saltutil.refresh_pillar`` for the client.
```

Comment:
Yes, it's correct, but maybe it makes sense to mention that in the other case it could be possible that `pillar.items` and `pillar.get` calls could produce different results with different values of pillars that were not refreshed,

Reply:
ok, done.